docs/cheaha/open_ondemand/hpc_desktop.md

```diff
@@ -85,7 +85,7 @@ To use this method you will need either a GitHub account or Microsoft account. M
 <!-- markdownlint-disable MD046 -->
 !!! warning
 
-    Do not use "Remote - SSH" to access Cheaha, as all processes run on the login node. VSCode Server, and associated processes, running on the login node may be shut down at any time to free login node resources. Instead, please use "Remote - Tunnels" as described below.
+    Please do not use IDEs like VSCode "Remote - SSH" or Cursor Server to connect to Cheaha. These methods run all of their processes on the login node, which are automatically terminated. Instead, please use "Remote - Tunnels" as described below.
```

Comment:
Good. Please also cross-link back to the original. This may benefit from macros in the future.

One small tweak, VSCode "Remote - SSH" is not an IDE, it's an extension. You could say "technologies" instead of IDEs. Instead of "which are automatically..." you might say "so they are automatically...", indicating that the processes on the login node is the reason why.

You may consider cross-linking to our "don't run on the login node" section/text, wherever that is. Same goes for the previous usage of this content.